### PR TITLE
docs: Add a link to docs/operator-manual/security.md from root-level security.md

### LINKED ISSUE #7833: docs: Add a link to docs/operator-manual/security.md from root-level security.md
# Summary

Add a link to docs/operator-manual/security.md from root-level security.md.

# Motivation

Reduce friction when users are trying to find in-depth security recommendations.

# Proposal

Add a new section to the root-level security.md with a brief sentence and link to the operator docs.

### PR DESCRIPTION
**Pull request by crenshaw-dev**
Closes #7833

Checklist:

* [x] Either (a) I've created an [enhancement proposal](https://github.com/argoproj/argo-cd/issues/new/choose) and discussed it with the community, (b) this is a bug fix, or (c) this does not need to be in the release notes.
* [x] The title of the PR states what changed and the related issues number (used for the release note).
* [x] I've included "Closes [ISSUE #]" or "Fixes [ISSUE #]" in the description to automatically close the associated issue.
* [ ] I've updated both the CLI and UI to expose my feature, or I plan to submit a second PR with them.
* [ ] Does this PR require documentation updates?
* [ ] I've updated documentation as required by this PR.
* [ ] Optional. My organization is added to USERS.md.
* [x] I have signed off all my commits as required by [DCO](https://github.com/argoproj/argoproj/tree/master/community#contributing-to-argo)
* [ ] I have written unit and/or e2e tests for my change. PRs without these are unlikely to be merged.
* [ ] My build is green ([troubleshooting builds](https://argo-cd.readthedocs.io/en/latest/developer-guide/ci/)).

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -64,3 +64,8 @@ findings (unless you prefer to stay anonymous, of course).
 Please report vulnerabilities by e-mail to the following address: 
 
 * cncf-argo-security@lists.cncf.io
+
+## Securing your Argo CD Instance
+
+See the [operator manual security page](docs/operator-manual/security.md) for 
+additional security information.
```
Comment (jannfis) [inline SECURITY.md:71]: Can we rephrase this a little bit please, e.g. to

> See the [operator manual security page](docs/operator-manual/security.md) for additional information about Argo CD's security features and how to make your Argo CD production ready.

**Review by jannfis — CHANGES_REQUESTED** [review]
Thanks @crenshaw-dev! I have one minor request for change (actually, very opinionated - open to discuss). Also, we do have a version in this doc. Please bump version number and date of change, thanks!

File: SECURITY.md (hunk 1 above)
Comment (crenshaw-dev) [reply SECURITY.md:71]: Yep, that's much better!

**Review by alexmt — APPROVED** [review]
LGTM

**Comment by alexmt** [comment]
@jannfis PTAL

**Review by jannfis — APPROVED** [review]
LGTM! Apologies, must have missed this one.

**Pull request merged**